### ISSUE TITLE
Expert mode as new normal?

### ISSUE THREAD

**Issue by rugk** [ENT8R/osm-microtasks#12]
I think the expert mode works great, maybe you can make it the "normal mode"?

**Comment by rugk** [comment]
Ah okay, you create one changeset for each change – that is probably not advisable. Thus this may need to be fixed first.

**Comment by ENT8R** [comment]
No, I wouldn't like to do that as the normal mode should be used by most users because this allows an extra validation step. Because all other values can be seen in the editor (e.g. iD) which is not the case if you are using the expert mode

> Ah okay, you create one changeset for each change – that is probably not advisable.

Mmh... I also don't know if it is better to have one changeset for every single change or have one changeset for multiple objects which might cover the whole state/country...

**Comment by rugk** [comment]
> I also don't know if it is better to have one changeset for every single change or have one changeset for multiple objects which might cover the whole state/country...

The same issue as in StreetComplete. I guess, for the same task for a specific area you may use the same changeset…
